src/rust/ide/src/controller/graph/executed.rs

```diff
@@ -21,10 +24,29 @@ impl Handle {
     /// This takes ownership of execution context which will be shared between all copies of this
     /// handle; when all copies will be dropped, the execution context will be dropped as well
     /// (and will then removed from LanguageServer).
-    pub fn new(graph:controller::Graph, execution_ctx:ExecutionContext) -> Self {
-        let execution_ctx = Rc::new(execution_ctx);
+    pub fn new(graph:controller::Graph, execution_ctx:Rc<ExecutionContext>) -> Self {
```

Comment:
Update the docs. The Executed Graph Controller no longer owns the execution_ctx, because the passed Rc can already have its copies somewhere.

Reply:
Done